runebender-lib/src/widgets/controller.rs

```diff
@@ -102,8 +102,8 @@ impl<W: Widget<EditorState>> Widget<EditorState> for EditorController<W> {
         let our_size = self.inner.layout(ctx, bc, data, env);
         let coords_size = self.coord_panel.layout(ctx, &child_bc, data, env);
         let coords_origin = (
-            (our_size.width / 2.0) - coords_size.width / 2.0,
-            our_size.height - coords_size.height - 20.0,
+            (our_size.width) - coords_size.width - 24.0,
```

Comment:
should these be using that `FLOATING_PANEL_PADDING` constant from above?

Reply:
Thanks! This is fixed in commit 2f238eb.